doc/api/http.md

```diff
@@ -2451,6 +2451,10 @@ header name:
   `etag`, `expires`, `from`, `host`, `if-modified-since`, `if-unmodified-since`,
   `last-modified`, `location`, `max-forwards`, `proxy-authorization`, `referer`,
   `retry-after`, `server`, or `user-agent` are discarded.
+  To allow duplicate values of the headers listed above to be joined,
```

Comment:
Can you also add the change and the pr url to the changes below the function in the docs?

Reply:
which version is this pr eventually gonna ship? v20.0.0?

Reply:
Use the value `REPLACEME` and the correct version will be automatically replaced when generating a release with this PR included.